cache/clock_cache.cc

```diff
@@ -73,6 +73,16 @@ inline void FreeDataMarkEmpty(ClockHandle& h, MemoryAllocator* allocator) {
   MarkEmpty(h);
 }
 
+// Called to undo the effect of referencing an entry for internal purposes,
+// so it should not be marked as having been used.
+inline void Unref(const ClockHandle& h, uint64_t count = 1) {
```

Comment:
Should this be non-const? I didn't get why this works fine

Reply:
The `meta` field is marked mutable so we can count read references in functions taking a const ClockHandle reference.